test-util/api-builder.js

```diff
@@ -0,0 +1,67 @@
+export function ApiBuilder(api) {
```

Comment:
Is this copied in from the request-baker project? Let's make a note to extract this into another package. Hapi-sov would have use for it as well :)